File: inutile/xx.pl
use strict;
use warnings;
use JSON;
use POSIX qw(strftime);

my @data;
for my $i (0..9) {
    push @data, { id => $i, value => int(rand(100)) };
}

my $json = JSON->new->pretty->encode(\@data);

open my $fh, '>', 'data.json' or die "Could not open file 'data.json' $!";
print $fh $json;
close $fh;

File: inutile/xx.ex
defmodule DataGenerator do
  def generate_data do
    data = for i <- 0..9 do
      %{"id" => i, "value" => :rand.uniform(100) - 1}
    end

    json = Jason.encode!(data, pretty: true)
    File.write!("data.json", json)
  end
end

DataGenerator.generate_data()